integrations/peframe/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.8.2-alpine3.11
+
+MAINTAINER eshaan7bansal@gmail.com
+
+RUN apk update && apk upgrade
+RUN apk add --no-cache git libssl1.1 swig g++ gcc make openssl-dev python3-dev libffi-dev libmagic 	
+
+# Build PEframe
+RUN git clone https://github.com/guelfoweb/peframe.git
+WORKDIR peframe/
```

Comment:
to easily understand where we can find the "additional" content of the container, please put peframe and the flask app in the same directory like `/opt/intel_owl` then chown it.